tools/scripts/scan-docker-images.sh

```diff
@@ -21,32 +21,54 @@ VULNERABILITIES_FOUND=3
 
 function print_usage_and_exit_with_error() {
   echo ${MAJOR_SEPARATOR}
-  echo Error: ${1}
+  echo "Error: ${1}"
   echo
   echo Usage:
-  echo 
-  echo "    ${0} path/to/bumpenvs.yaml"
+  echo
+  echo "    ${0} {path/to/bumpenvs.yaml | --images [image_name ...]}"
   echo ${MAJOR_SEPARATOR}
   exit ${INVALID_ARGS}
 }
 
-if [ ${#} -ne 1 ]; then
-  print_usage_and_exit_with_error "${#} arguments received, expected 1"
-fi
-
-if [ ! -f ${1} ]; then
-  print_usage_and_exit_with_error "File ${1} does not exist"
+if [ ${#} -lt 1 ]; then
+  print_usage_and_exit_with_error "${#} arguments received, expected 1 or more"
 fi
 
-BUMPENVS=${1}
+while [[ $# -gt 0 ]]; do
```

Comment:
Being incredibly nitpicky here: $# -> ${#} for consistent style. Not worth resubmitting the PR unless we update this for another reason anyway, IMO.